humble/distribution.yaml

```diff
@@ -578,6 +578,72 @@ repositories:
       url: https://github.com/pal-robotics/aruco_ros.git
       version: humble-devel
     status: developed
+  as2_platform_crazyflie:
+    doc:
+      type: git
+      url: https://github.com/aerostack2/as2_platform_crazyflie.git
+      version: main
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/aerostack2/as2_platform_crazyflie.git
+      version: main
+    status: developed
+  as2_platform_dji_osdk:
```

Comment:
Changed repo so running checks again:

- [x] At least one of the following must be present
  - [x] Top level license file: https://github.com/aerostack2/as2_platform_dji_osdk/blob/main/LICENSE
  - [ ] Per package license files: 
- [x] License is [OSI-approved](https://opensource.org/licenses): [BSD 3-Clause ](https://opensource.org/license/bsd-3-clause)
- [x] License correctly listed in package.xmls
- [x] Public source repo: https://github.com/aerostack2/as2_platform_dji_osdk
- [x] Source repository contains ROS packages
- [x] Each package meets [REP-144](https://www.ros.org/reps/rep-0144.html) naming conventions

<details><summary>Package name details</summary>

```console
$ find . -name "package.xml" -exec grep --color=auto -e "<name>" "{}" ";"
  <name>as2_platform_dji_osdk</name>
```
</details>

<details><summary>License details</summary>

```console
$ find . -name "package.xml" -exec grep --color=auto -e "<license>" "{}" "+"
  <license>BSD-3-Clause</license>


```
</details>